sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -102,4 +102,11 @@ export interface StudentFileUploaderDto {
 export interface StudentUploadFileDto {
   fileName: string;
   uniqueFileName: string;
+  metadata: StudentFileMetadataDTO;
+  groupName: string;
```

Comment:
since these added values are not required for student file summary, please create or extend a different dto for ministry user

Reply:
As there is no sensitive information for the student to see, so reusing it.

Reply:
It is not about sensitive information. in this case, data are clearly different for aest and student. and you have 2 controllers, I suggest keeping 2 different DTOs or extend this DTO

Reply:
I think that it would be ok for this particular case, but we need always keep in mind what @ann-aot mentioned.

Reply:
If that is ok for this particular scenario, there is a conflict in interpretation of mandatory fields because that changes as per the API using this DTO. 
May be I am missing something here.